### DISCUSSION TITLE
Popup Menu Default Selection

### DISCUSSION THREAD

**Discussion by Petri-Oosthuizen** [primefaces/community#1123] [PrimeVue]
Consider adding a toggle to prevent the first menu item from being highlighted by default. This has caused confusion for some users, as dropdown selectors typically only appear after using arrow keys.

![image](https://github.com/primefaces/community/assets/4954611/d5c4b75b-a001-491d-87cd-3e4620f10e4f)

**Comment by SagewLivingstone** [comment]
Yeah, this seems like a bug, happening for me too. This behavior does not happen in the docs, which makes me think its unintended.
